packages/dds/merge-tree/src/sequencePlace.ts

```diff
@@ -55,6 +55,23 @@ export enum Side {
 	After = 1,
 }
 
+/**
+ * Returns the given place in InteriorSequencePlace form.
+ * @alpha
+ */
+export function normalizePlace(place: SequencePlace): InteriorSequencePlace {
```

Comment:
i'm not sure we should add this, in generally nothing in the merge tree package should deal with sequence place, only numbers and InteriorSequencePlace